src/ApplicationCore/Entities/FraudProtectionApiModels/Purchase.cs

```diff
@@ -506,77 +189,52 @@ public class PurchaseProduct
         /// <summary>
         /// 	Indicates if the product is offered for preorder.
         /// </summary>
-        public Boolean IsPreorder { get; set; }
+        public Boolean? IsPreorder { get; set; }
 
         /// <summary>
         /// 	Indicates the method used to ship the product.
         /// </summary>
         public String ShippingMethod { get; set; }
-
-        /// <summary>
-        /// 	Product details associated with this product
-        /// </summary>
-        public Product ProductDetails { get; set; }
     }
 
     /// <summary>
-    /// 	Product details associated with this product
+    /// The possible product types
```

Comment:
nit: some of these summaries have the inner stuff tabbed in, some don't.

Reply:
Thanks, fixed in the next push.